model/data_test.go

```diff
@@ -0,0 +1,24 @@
+package model
+
+import (
+	"testing"
+)
+
+func TestAddReflogHash(t *testing.T) {
+
+	gci := SetGitExecuter(testExecuter{}) // use mock for unit test
+	gci.AddReflogHash()
+
+	got := len(gci.ReflogHashList)
+	want := 6
+	if got != want {
+		t.Fatalf("want %v, but %v:", want, got)
+	}
+}
+
+// testExecuter is mock of exuecuter
+type testExecuter struct{}
```

Comment:
It is preferable to use a name that is a pair of `GitCommanders` and that knows to be mock

Reply:
I agree with you. I changed struct name to `GitCommanderMock`.